src/services/FileUploadService.js

```diff
@@ -0,0 +1,22 @@
+import axios from "axios";
+
+const upload = (file, onUploadProgress)=>{
+    let formData = new FormData();
+    
+    formData.append("file", file);
+
+    return axios.post("users/upload", formData, {
```

Comment:
Will this hit the correct host?

Reply:
this needs to be specified to http://localhost:5000/users/upload